src/mongoose_http_handler.erl

```diff
@@ -85,3 +88,13 @@ configurable_handler_modules() ->
      mongoose_client_api,
      mongoose_admin_api,
      mongoose_graphql_handler].
+
+%% @doc Call instrumentation for a module of `mongoose_http_handler' behaviour
+-spec instrumentation(module()) -> [mongoose_instrument:spec()].
+instrumentation(Module) ->
+    case erlang:function_exported(Module, instrumentation, 0) of
```

Comment:
This wouldn't work if module is not loaded. I know it shouldn't be the case, but in general we are using `mongoose_lib:is_exported`.